### ISSUE TITLE
Jenkins test failures in tune.py.

### ISSUE THREAD

**Issue by robertnishihara** [ray-project/ray#1189]
The Jenkins tests seem to be failing frequently with

```
usage: tune.py [-h] [--redis-address REDIS_ADDRESS] [--num-cpus NUM_CPUS]
               [--num-gpus NUM_GPUS] -f CONFIG_FILE
tune.py: error: unrecognized arguments: --env mnist --stop {"training_iteration": 2}
```

See https://amplab.cs.berkeley.edu/jenkins/job/Ray/2114/console for example.

Any ideas @ericl @richardliaw?

**Comment by richardliaw** [comment]
So the culprint line is here:
https://github.com/ray-project/ray/blob/52888e4c6fe6bb9e1a364383060b1f4d58f459f9/test/jenkins_tests/run_multi_node_tests.sh#L131

I think the right fix should we keep a `tune` test is just to change it to 
```python /ray/python/ray/tune/examples/tune_mnist_ray.py```?

Alternatively, I don't think we support CLI experiments anymore, so maybe it's fine to remove.

@ericl

**Comment by robertnishihara** [comment]
When you say "we don't support CLI experiments anymore", you mean that users are supposed to use the Python hyperparameter tuning API and not the `tune.py` script, right?

And internally `tune_mnist_ray.py` uses the Pytho hyperparameter tuning API, right?

If that's the case, then let's just do

```
docker run --shm-size=10G --memory=10G $DOCKER_SHA \ 
     python /ray/python/ray/tune/examples/tune_mnist_ray.py
```

or something like that, which is what you are suggesting I think?

**Comment by richardliaw** [comment]
Ah sorry. I meant that there is still a CLI, but not one like before. Now,
`tune.py` can only be used either with a YAML/JSON file, rather than
specifying all the experiment parameters in a series of flags.
Alternatively, users can use the Python API.

`tune_mnist_ray.py` uses the Python API internally, and the latter
suggestion is correct.

**Comment by robertnishihara** [comment]
Fixed by #1194.
